index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+    <h1>Genotify</h1>
+    <form>
+        Search for:
+        <br>
+        <input type="text" id="query" placeholder="CDK2, cyclin kinase 2, etc">
+    </form>
+    <br>
+    <button id="searchButton" type="button">Search</button>
+    <p id="basics"></p>
+    <br>
+    <p id="summary"></p>
+    <script>
```

Comment:
There's nothing wrong with including JavaScript within an HTML file. What I said to avoid was stuff like this:

```html
<a href="#" onclick="console.log('This is some JavaScript');">Click Me</a>
```

Reply:
Oh, yeah, inline stuff is awful.